docker-images/baleine/Dockerfile

```diff
@@ -1,5 +1,28 @@
 FROM ubuntu:latest
 
+# Install necessary binaries for tutorials
+RUN apt update -y && apt upgrade -y && apt install kmod iw iputils-ping openssh-client openssh-server bash netcat iproute2 net-tools tcpdump dnsutils netcat nano vim -y
+
+# Setup Bash prompt to not get lost when in interactive mode
+RUN echo "PS1='[CONTAINER] ${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '" | tee /root/.bashrc > /dev/null
+
+#Setting up OpenSSH server
+RUN sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config
+RUN sed -i 's/#Port 22/Port 2222/g' /etc/ssh/sshd_config
+RUN sed -i 's/#ListenAddress 0.0.0.0/ListenAddress 0.0.0.0/g' /etc/ssh/sshd_config
+RUN echo "auth sufficient pam_permit.so" | tee -a /etc/pam.d/sshd >> /dev/null
+
+RUN mkdir -p /run/sshd
+RUN echo "PermitEmptyPasswords yes" | tee -a /etc/ssh/sshd_config >> /dev/null
```

Comment:
again, why the tee ?

Reply:
well, it would do that if its output wasn't redirected to /dev/null
yes please get rid of all the tee's that are redirected to /dev/null

Reply:
After checking the command closer, the `tee -a` appends the `PermitEmptyPasswords yes` to /etc/ssh/sshd_config, my bad.